### PR TITLE
Enforcing style for parameter wrapping

### PR DESCRIPTION
**Pull request by kyle-long**

### CODE REVIEW

File: lang/py/index.md [hunk 1]
```diff
@@ -59,5 +59,18 @@ Do not use [string.format](https://docs.python.org/2/library/stdtypes.html#str.f
...
+
+    # GOOD
+    my_func_with_lots_of_parameters(
+        param1
+        param2
+        param3
+        param4
+    )
+
+    # BAD
```
Comment (AbsentSemicolon) [inline lang/py/index.md]: I was caught on this too, we don't want to show bad, just what should be done and are there comma between params?

**Review by AbsentSemicolon — CHANGES_REQUESTED** [review]

**Comment by fidian** [comment]
Looks good, but I agree that we want to avoid the examples showing negatives.  We could show it dozens of ways that are wrong and I don't want to maintain that list.  Also, when you include bad examples then people will say, "but I totally saw it that way."

**Review by fidian — APPROVED** [review]
Yay!

**Review by AbsentSemicolon — APPROVED** [review]

**Comment by AbsentSemicolon** [comment]
Looks good.

**Pull request merged**